price-feeder/config/config.go

```diff
@@ -200,6 +201,17 @@ func ParseConfig(configPath string) (Config, error) {
 		}
 	}
 
+	for quote := range coinQuotes {
+		for index, pair := range cfg.CurrencyPairs {
+			if pair.Base == quote && pair.Quote == denomUSD {
+				break
+			}
+			if index == len(cfg.CurrencyPairs)-1 {
+				return cfg, fmt.Errorf("all non-usd quotes require a conversion rate feed")
```

Comment:
Please note that since certain stablecoins may not have at least three providers for converting to USD, we shouldn't cut a new release of the price feeder until #792 is addressed